### ISSUE TITLE
Bulk image upload

### ISSUE THREAD

**Issue by bethlakshmi** [bethlakshmi/abc-inventory#15]
With the ABC Docents (ABCDs?) valiantly scanning our pictures, I want a way to make upload easy.

My plan:

1 - a Button on the list items page for "Upload Images"
2 - it takes you to a prompt for a bulk upload.  I'll investigate if there are better multi-file uploads than the default. But not if it takes too long.  My concern is that big files uploads can be slow, and the big sites that do that (drop box, google, etc) - have a UI with info and a progress bar (and still it gives me problems...) - if there's an easy client side library to help here, I'll do it.  If not, ya on ya own and it's a separate ticket.
3 - it's a wizard, so after succcesful upload, user gets a list of items (with thumbnails) that they can associate with any item.  Each image can be associated with exactly 1 or 0 items on this form.   To do multiple item associations, the user would have to use the item -> image association (see #12).  To make a new item, the user will have to use the "Create Item" button already on the list page and then #12

Once (2) is done, the items are uploaded.  Once (3) is done, the item is configured.

**Comment by bethlakshmi** [comment]
Deployed on live this morning.
